scripts/deploy-turbopack-docs.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+
+PROD=""
+
+if [ -z ${DEPLOY_ENVIRONMENT+x} ]; then
+    DEPLOY_ENVIRONMENT=""
+else
+    if [ "$DEPLOY_ENVIRONMENT" = "production" ]; then
+      PROD="--prod"
+    fi
+fi
+
+RUSTDOCFLAGS="-Z unstable-options --index-page $(pwd)/packages/next-swc/docs/index.md" cargo doc -p turbo-tasks-fs --no-deps
```

Comment:
Why `turbo-tasks-fs`?

Reply:
only pkg I can casually find to build with rustdoc without 1. huge time to check CI 2. does not actually raises build error.

Reply:
yes, there's no point for doc itself. It's setup that ensures

1. we can build cargo docs without build error
2. we can deploy it correctly.

It's just starting point to actually add docs.